parser/GBenchToASV.py

```diff
@@ -0,0 +1,134 @@
+import os
+import sys
+import json
+import subprocess
+import argparse
+import re
+import platform
+import psutil
+
+from asvdb import ASVDb, BenchmarkInfo, BenchmarkResult
+
+# USAGE:
+#
+#   -d : JSON Result Directory
+#   -n : Repository Name
+#   -t : Target Directory for ASV JSON
+#   -b : Branch Name
+
+def build_argparse():
+    parser = argparse.ArgumentParser(add_help=True)
+    parser.add_argument('-d', nargs=1, help='JSON Result Directory')
+    parser.add_argument('-n', nargs=1, help='Repository Name')
+    parser.add_argument('-t', nargs=1, help='Target Directory for JSON')
+    parser.add_argument('-b', nargs=1, help='Branch Name')
+    return parser
+
+
+def getCommandOutput(cmd):
+    result = subprocess.run(cmd,
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE,
+                            shell=True)
+    stdout = result.stdout.decode().strip()
+    if result.returncode == 0:
+        return stdout
+
+    stderr = result.stderr.decode().strip()
+    raise RuntimeError("Problem running '%s' (STDOUT: '%s' STDERR: '%s')"
+                       % (cmd, stdout, stderr))
+
+
+def getSysInfo():
+    uname = platform.uname()
+    commitHash = getCommandOutput("git rev-parse HEAD")
+    commitTime = getCommandOutput("git log -n1 --pretty=%%ct %s" % commitHash)
+    commitTime = str(int(commitTime)*1000)  # ASV wants commit to be in milliseconds
+
+    bInfo = BenchmarkInfo(
+                machineName=uname.machine,
```

Comment:
I realized that `uname.machine` didn't give me what I wanted for my jobs (eg. on my machine it just says `x86_64` instead of `rratzel-Z8`). I think you can get the name users expect by either using `uname.node` or by allowing the Jenkins job to pass it through via a new CLI arg. On my machine, `uname.node` returns `rratzel-Z8`.  I don't know what this returns on an EC2 instance, so allowing the job to pass in the machine name it's working with (which users probably understand) might be best. Your call.